drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-api/src/main/java/org/drools/workbench/screens/dtablexls/service/DecisionTableXLSService.java

```diff
@@ -44,4 +45,7 @@ Path create( final Path resource,
                  final String sessionId,
                  final String comment );
 
+    Path saveMetadata(final Path resource,
```

Comment:
third time?

Reply:
We also have the `rename` and `saveAndRename` copied in the same way. I think I'll move this `saveMetadata` to a remote service so that the button calls it directly. For `rename` and `saveAndRename` I'll make a ticket where the plan is to do the same.

Reply:
https://issues.jboss.org/browse/DROOLS-4608

Reply:
Also the save method should be removed for upload widget based editors https://issues.jboss.org/browse/DROOLS-4609